instrumentation/runtime-metrics/javaagent/src/test/groovy/io/instrumentation/javaagent/instrumentation/runtimemetrics/RuntimeMetricsTest.groovy

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.instrumentation.javaagent.instrumentation.runtimemetrics
+
+import io.opentelemetry.instrumentation.test.AgentInstrumentationSpecification
+import spock.util.concurrent.PollingConditions
+
+class RuntimeMetricsTest extends AgentInstrumentationSpecification {
+
+  def "test runtime metrics is enabled"() {
+    when:
+    def conditions = new PollingConditions(timeout: 10, initialDelay: 1.5, factor: 1.25)
+
+    then:
+    conditions.eventually {
+      assert metrics.any { it.name == "runtime.jvm.gc.collection" }
```

Comment:
Where does this `metrics` come from?

Reply:
`InstrumentationSpecification`

Reply:
I had the exact same confusion when reading SystemMetricsTest. I renamed these to `getMetrics()` since I think it's less mysterious.